spec/features/login_spec.rb

```diff
@@ -1,21 +1,23 @@
 require "rails_helper"
 
 RSpec.describe "Logging in with slack", type: :feature do
-  let(:user) { build(:user) }
+  let(:slack_id) { "raw_slack_id_12345" }
+  let(:hashed_id) { Digest::SHA256.hexdigest(slack_id) }
+  let(:user) { create(:user, slack_id: hashed_id) }
 
   before do
     OmniAuth.config.test_mode = true
     OmniAuth.config.mock_auth[:slack] = OmniAuth::AuthHash.new(
       info: {
-        user_id: user.slack_id
+        user_id: slack_id
       }
     )
   end
 
   scenario "shows logout link" do
     visit root_path
     click_link
-    expect(page).to have_content(user.slack_id)
+    expect(page).to have_content(hashed_id)
```

Comment:
Maybe this is post-MVP, but I'm wondering if it'd make sense to display a user's name instead of their slack id. I believe there's a way to access the info via omniauth without saving it in the db, right? Though I'm not sure if that'd be pushing it on any API call limits we might be working with.

Reply:
My thought is that showing the name undermines the anonymity. We do have access to their name when they go through slack oauth, but IMO the only thing we should get is that it is a user from Turing slack.